de.dlr.sc.overtarget.language/src/de/dlr/sc/overtarget/language/generator/OvertargetOutputConfigurationProvider.java

```diff
@@ -44,6 +49,17 @@
 		
 		outputConfigurations.add(conceptOutput);
```

Comment:
I just noticed that the regular output configuration is called conceptOutput.
This was probably copied over from VirSat. We should change the name to something with overtarget
to make it clear what this configuraiton is.

Reply:
Fixed.